src/rebuild/srv.c

```diff
@@ -1581,8 +1587,7 @@ rebuild_leader_stop(const uuid_t pool_uuid, unsigned int ver, unsigned int gen,
 	d_list_for_each_entry_safe(task, task_tmp, &rebuild_gst.rg_queue_list,
 				   dst_list) {
 		if (uuid_compare(task->dst_pool_uuid, pool_uuid) == 0 &&
-		    (ver == (unsigned int)(-1) || task->dst_map_ver == ver) &&
-		    (gen == (unsigned int)(-1) || task->dst_rebuild_gen == gen))
```

Comment:
Since " pool_uuid + map version + leader term + rebuild generation define a rebuild job", then why not check "gen" and "term" when destroy the rebuild task?

Reply:
oh, "gen" and "term" are only for running rebuild task, i.e. they are not defined in dst_task, so we do not need check them.